pkg/kubelet/kubelet.go

```diff
@@ -1573,26 +1573,130 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 	}
 }
 
-// Updated the Kubelet's internal pods with those provided by the update.
+func isStaticPod(pod *api.BoundPod) bool {
+	if getPodSource(pod) != ApiserverSource {
```

Comment:
Sorry, I should describe that more :) What are the options for pod source? We have HTTP, static, and API server. Are we ignoring HTTP here as well?